onelive/archive/index.html

```diff
@@ -34,19 +34,34 @@
 
 <main>
     <!--main content goes here-->
-    <section style="background: url('../images/header-onelive.svg') right bottom no-repeat; height: 70vh">
-        <!--Navbar is placed here to align the background image correctly-->
+    <section class="section">
+        <!--navbar goes here-->
```

Comment:
Some early commits have been added to this PR, that's because you have pushed it to the master without resetting the branches. So you have to do like below,

First in the clone your forked repo,
- Go to that folder by cd FILE_NAME
- git remote add upstream https://github.com/sef-global/sef-site.git
- Then create a new branch by git branch BRANCH_NAME
- Then checkout it by git checkout BRANCH_NAME
- After that fetch the upstream by git fetch upstream
- Reset and pull by git reset --hard upstream/master , git pull upstream master
- Do the changes, add and commit them by old way
- Then push the changes as git push origin BRANCH_NAME

Reply:
> Some early commits have been added to this PR, that's because you have pushed it to the master without resetting the branches. So you have to do like below,
> 
> First in the clone your forked repo,
> 
> * Go to that folder by cd FILE_NAME
> * git remote add upstream https://github.com/sef-global/sef-site.git
> * Then create a new branch by git branch BRANCH_NAME
> * Then checkout it by git checkout BRANCH_NAME
> * After that fetch the upstream by git fetch upstream
> * Reset and pull by git reset --hard upstream/master , git pull upstream master
> * Do the changes, add and commit them by old way
> * Then push the changes as git push origin BRANCH_NAME

I will do this yoshitha

Reply:
Do this for all the PRs you put before pushing

Reply:
> Do this for all the PRs you put before pushing

Sure

Reply:
![image](https://user-images.githubusercontent.com/88109775/204243933-b19138cd-70db-4cb3-b556-5ae8be4f105a.png)

I think we don't have all the details about some of our mentees

Reply:
> > ![image](https://user-images.githubusercontent.com/88109775/204243933-b19138cd-70db-4cb3-b556-5ae8be4f105a.png)
> > I think we don't have all the details about some of our mentees
> 
> I can't see any images in mentees @janithlahirukariyawasam, how did you got the other images?

**This type doesn't contain information about mentees' uni and profile photo**
![api 1](https://user-images.githubusercontent.com/88109775/204356119-f7e3e355-170e-41c4-a734-9ca4ac20592f.png)

**This type contains uni and picture details also.**
**please open the image in a new window to show details.**
![api 2](https://user-images.githubusercontent.com/88109775/204356183-153383f2-55a2-4c50-8df5-01fa912f434f.png)

Reply:
@janithlahirukariyawasam There are some missing data. For now let's implement the functionality

Reply:
> > > ![image](https://user-images.githubusercontent.com/88109775/204243933-b19138cd-70db-4cb3-b556-5ae8be4f105a.png)
> > > I think we don't have all the details about some of our mentees
> > 
> > 
> > I can't see any images in mentees @janithlahirukariyawasam, how did you got the other images?
> 
> **_In the Above pull request I didn't add the profile photos and uni data_** It only consists with names and mentors. ![search ](https://user-images.githubusercontent.com/88109775/204358162-31e0523d-dc23-4529-84c6-5e09d854585d.png)
> 
> **_This is the planned implementation_** ![image](https://user-images.githubusercontent.com/88109775/204358687-bb3754d3-7cc9-49c3-aef6-669aaf0bc7a9.png)
> 
> But , Actually, I think there are two types of mentee data we have, one set with images/uni details & etc one set with names/mentors.
> 
> **This type doesn't contain information about mentees' uni and profile photo** ![api 1](https://user-images.githubusercontent.com/88109775/204356119-f7e3e355-170e-41c4-a734-9ca4ac20592f.png)
> 
> **This type contains uni and picture details also.** **please open the image in a new window to show details.** ![api 2](https://user-images.githubusercontent.com/88109775/204356183-153383f2-55a2-4c50-8df5-01fa912f434f.png)
> 
> If we going to implement planned implementation some rows will not contain any data relevant to profile photo and university. @YoshithaRathnayake **_I didn't add the profile photo and uni data to above pull request_**

Ok then